auth.mkd

```diff
@@ -455,68 +431,68 @@ DRIP Link is important as its contents are used to provide trust in the DET/HI p
 |                                                               |
 |                                                               |
 +---------------+---------------+---------------+---------------+
+~~~~
+{:fig #link-fig title="Broadcast Endorsement / DRIP Link"}
 
-VNB Timestamp by Parent (4 bytes):
-    Current time at signing, set by Parent Entity.
+VNB Timestamp by Parent (4 octets):
 
-VNA Timestamp by Parent (4 bytes):
-    Timestamp denoting recommended time to trust Endorsement.
+> See {{astm-timestamps}}. Set by Parent Entity.
 
-DET of Child: (16 bytes)
-    DRIP Entity Tag of Child Entity.
+VNA Timestamp by Parent (4 octets):
 
-HI of Child: (32 bytes)
-    Host Identity of Child Entity.
+> See {{astm-timestamps}}. Set by Parent Entity.
 
-DET of Parent: (16 bytes)
-    DRIP Entity Tag of Parent Entity.
+DET of Child: (16 octets)
 
-Signature by Parent(64 bytes):
-    Signature over preceding fields using the keypair of
-    the Parent DET.
-~~~~
-{:fig #link-fig title="Broadcast Endorsement / DRIP Link"}
+> DRIP Entity Tag of Child Entity.
+
+HI of Child: (32 octets)
+
+> Host Identity of Child Entity.
+
+DET of Parent: (16 octets)
+
+> DRIP Entity Tag of Parent Entity in DIME Hierarchy.
+
+Signature by Parent(64 octets):
+
+> Signature over concatenation of preceding fields (`VNB`, `VNA`, `DET of Child`, `HI of Child`, and `DET of Parent`) using the keypair of the Parent DET.
 
 This DRIP Authentication Message is used in conjunction with other DRIP SAM Types (such as the Manifest or the Wrapper) that contain data (e.g., the ASTM Location/Vector Message, Message Type 0x2) that is guaranteed to be unique, unpredictable, and easily cross-checked by the receiving device.
 
-A hash of the final link (`Broadcast Endorsement: HDA on UA`) in the Broadcast Endorsement chain SHOULD be included in each DRIP Manifest {{drip-manifest}}.
+A hash of the final link (`BE: HDA on UA`) in the Broadcast Endorsement chain SHOULD be included in each DRIP Manifest {{drip-manifest}}.
```

Comment:
When this "SHOULD" can be safely ignored? If you that have any case, then consider MUST

Reply:
Made a MUST